.github/workflows/build-and-test-pr.yaml

```diff
@@ -36,6 +36,14 @@ jobs:
           target-packages: ${{ steps.get-modified-packages.outputs.modified-packages }}
           build-depends-repos: build_depends.repos
 
+      - name: Upload coverage to Codecov
+        if: ${{ steps.get-modified-packages.outputs.modified-packages != '' }}
+        uses: codecov/codecov-action@v2
+        with:
+          files: lcov/total_coverage.info,coveragepy/.coverage
+          fail_ci_if_error: true
+          verbose: true
```

Comment:
Is it possible to edit some files and test this in this PR?

Reply:
OK. I will test this workflow by adding a new commit.

Reply:
Thank you!
https://github.com/autowarefoundation/autoware.universe/pull/292#issuecomment-1020797172